Scripts/validate-parlamint.xsl

```diff
@@ -159,8 +159,8 @@
     <xsl:variable name="title-prefix">[^ ]+( [^ ]+)? parliamentary corpus <xsl:value-of select="$idTemplate"/></xsl:variable>
     <xsl:variable name="title-suffix">
       <xsl:choose>
-        <xsl:when test="/tei:teiCorpus and $type = 'txt'"> \[ParlaMint( SAMPLE)?\]$</xsl:when>
-        <xsl:when test="/tei:teiCorpus and $type = 'ana'"> \[ParlaMint\.ana( SAMPLE)?\]$</xsl:when>
+        <xsl:when test="not(/tei:TEI) and $type = 'txt'"> \[ParlaMint( SAMPLE)?\]$</xsl:when> <!-- teiHeader context when testing teiCorpus header -->
```

Comment:
@TomazErjavec title was not validated correctly since header factorization changes. This fixes it

Reply:
Not sure I understand what happened here, but I am sure you do, so, thanks!